packages/IMAPClient-Core.package/ICEndPoint.class/class/fetchHeadersWithUIDRange.to..st

```diff
@@ -3,4 +3,4 @@ fetchHeadersWithUIDRange: aBeginId to: anEndId
 
 	^ {self protocolCommandStart . 'UID FETCH' .
 		(aBeginId asString, ':', anEndId asString) .
-		'(body[HEADER.FIELDS (from to subject date)])' } joinSeparatedBy: self protocolBlank
\ No newline at end of file
+		'(BODY.PEEK[HEADER.FIELDS (from to subject date)])' } joinSeparatedBy: self protocolBlank
```

Comment:
If you change `body` into `BODY.PEEK` the tests are going to fail. Because they mock a MailServer, which got hardcoded rules on how to reply to requests. This broke the tests here. So please check whether the test on travis are working after you made your pull request, and if not, try to fix it :) @ClayGee fixed it here. Just for the next time :)